docs/excel/excel-data-types-concepts.md

```diff
@@ -14,17 +14,30 @@ ms.localizationpriority: high
 
 This article describes how to use the [Excel JavaScript API](../reference/overview/excel-add-ins-reference-overview.md) to work with data types. It introduces core concepts that are fundamental to data type development.
 
-## Core concepts
+## The `valuesAsJson` property
 
-Use the [`Range.valuesAsJson`](/javascript/api/excel/excel.range#excel-excel-range-valuesasjson-member) property to work with data type values. This property is similar to [Range.values](/javascript/api/excel/excel.range#excel-excel-range-values-member), but `Range.values` only returns the four basic types: string, number, boolean, or error values. `Range.valuesAsJson` returns expanded information about the four basic types, and this property can return data types such as formatted number values, entities, and web images.
+The `valuesAsJson` property is integral to creating data types in Excel. This property is an expansion of the `values` property available on multiple Excel JavaScript API objects, like [Range.values](/javascript/api/excel/excel.range#excel-excel-range-values-member). Both the `values` and `valuesAsJson` properties are used to access the value in a cell, but the `values` property only returns one of the four basic types: string, number, boolean, or error values. In contrast, `valuesAsJson` returns expanded information about the four basic types, and this property can return data types such as formatted number values, entities, and web images.
```

Comment:
Also, I'm not sure `values` returns an error. According to the existing docs, "Cells that contain an error will return the error string."

Reply:
Interesting point. It doesn't return an API error (such as `InvalidArgument`), it returns the error value in the cell as a string, such as #NUM!. The data types articles refer to the four basic types this way elsewhere too, with error in the list (see [Data types overview](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-data-types-overview)). 
 
Do you think it would be clearer here if the sentence said "error data type" instead of "error value"?: 

> Both the `values` and `valuesAsJson` properties are used to access the value in a cell, but the `values` property only returns one of the four basic types: string, number, boolean, or error data types.